### DISCUSSION TITLE
symlinked files

### DISCUSSION THREAD

**Discussion by DoctorSubtilis** [Zettlr/Zettlr#3457] [General]
I found yesterday Zettlr very interesting (probably the best md editor).  
So I wonder if it is my fault if I can't open symlinked files (neither within the app nor from an external file manager).  
Since I use Linux, symlinking a file is a very common (and useful) practice.  
There is a workaround, or I should do something?
Thank you!

**Comment by kyaso** [comment]
No, symlinks are not supported. Feel free to open an issue!

**Reply by nathanlesage** [reply]
> No, symlinks are not supported. Feel free to open an issue!

There is already a pinned issue calling for symlink support!
